docs/ref/logging.txt

```diff
@@ -226,7 +226,7 @@ a mail sending exception.
 
 Log messages related to :doc:`contrib/gis/index` at various points: during the
 loading of external GeoSpatial libraries (GEOS, GDAL, etc.) and when reporting
-errors. Each ``ERROR`` log record includes the caught exception and relevant
+errors. Each ``WARNING`` log record includes the caught exception and relevant
```

Comment:
Not all errors have become warnings, there are other errors logged within `django/contrib/gis/`.
I think each of those should be checked as to whether they should still be errors (e.g. the other error logged in `django/contrib/gis/forms/widgets.py`) and we should be clear when errors are logged and when there are warnings
I also think maybe the change could be a debug message but that might become clearer when explaining the message types